com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/HDGlobalSettingsWindow.cs

```diff
@@ -455,14 +455,20 @@ static void DrawMiscSettings(SerializedHDRenderPipelineGlobalSettings serialized
                 if (serialized.useDLSSCustomProjectId.boolValue)
                     EditorGUILayout.PropertyField(serialized.DLSSProjectId, Styles.DLSSProjectIdLabel);
 #endif
+
+                Type renderPipeManagerType = typeof(RenderPipelineManager);
+                var cleanupRenderPipeline = renderPipeManagerType.GetMethod("CleanupRenderPipeline", BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public);
                 EditorGUI.BeginChangeCheck();
                 EditorGUILayout.PropertyField(serialized.supportProbeVolumes, Styles.probeVolumeSupportContentLabel);
                 if (EditorGUI.EndChangeCheck())
                 {
                     // If we are running HDRP, we need to make sure the RP is reinitialized
                     if (HDRenderPipeline.currentPipeline != null)
```

Comment:
Was that supposed to be removed since we seem to be calling it in a more generic way now?

Reply:
what generic way are you talking about?

Reply:
The part where we get the method like this GetMethod("CleanupRenderPipeline"). Or am I misunderstanding things?
If not then I don't understand why this was added.

Reply:
actually it is a bad merge - i moved that code out of this function. @arttu-peltonen could you double check

Reply:
Whoops yes merge artifact indeed. Fixed now - thanks @JulienIgnace-Unity and @jenniferd-unity!